### PR TITLE
typo and grammar scrub

### PR DESCRIPTION
**Pull request by avarsava**
Incidentally, fixes a few bugs in guided meditations!

JIRA Ticket: 

- [ ] Updates Changelog
- [ ] Updates developer documentation

### CODE REVIEW

File: shesmu-server-ui/src/histogram.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export interface HistogramRow {
    */
   selectionDisplay(start: number, end: number): string;
   /**
-   * A callback inidcating that the user has selected this range of the histogram.
+   * A callback inidicating that the user has selected this range of the histogram.
```
Comment (mlaszloffy) [inline shesmu-server-ui/src/histogram.ts:28]:
```suggestion
   * A callback indicating that the user has selected this range of the histogram.
```

File: shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/plugins/PluginManager.java [hunk 2]
```diff
@@ -1889,7 +1889,8 @@ public static CallSite bootstrapServices(
               collector,
               callsite.dynamicInvoker().asType(MethodType.methodType(RequiredServices.class)));
     }
-    // Now, we should have a method thats () → String[], which is what we wanted, so shove it in the
+    // Now, we should have a method that's () → String[], which is what we wanted, so shove it in
+    // the
```
Comment (mlaszloffy) [inline shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/plugins/PluginManager.java:1893]:
```suggestion
    // the olive
```
and delete line below?

**Review by mlaszloffy — APPROVED** [review]

File: shesmu-server-ui/src/histogram.ts (hunk 1 above)
Comment (rnaioicr) [inline shesmu-server-ui/src/histogram.ts]: "indicating"

File: docs/actnow.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Static Actions
-While Shesmu is mostly about the olives, it _does_ runs actions and it can be
+While Shesmu is mostly about the olives, it _does_ run actions and it can be
```
Comment (rnaioicr) [inline docs/actnow.md:2]: might be nitpicky, but maybe "While Shesmu is mostly about olives" vs "While Shesmu is mostly about the olives"

**Review by rnaioicr — COMMENTED** [review]
Just a couple adjustments!

File: shesmu-server-ui/src/histogram.ts (hunk 1 above)
Comment (avarsava) [reply shesmu-server-ui/src/histogram.ts]: Fixed in 48916bcb8b640282890da1e894e5635bad6c3531

**Review by rnaioicr — APPROVED** [review]

**Pull request merged**